Library/Formula/swiftgen.rb

```diff
@@ -14,33 +14,34 @@ class Swiftgen < Formula
   depends_on :xcode => "7.0"
 
   def install
-    ENV["SWIFTGEN_VERSION"] = "#{version} (via Homebrew)"
-    rake "install[#{bin},/]"
+    rake "install[#{bin},#{lib}]"
 
     fixtures = %w[
-      Tests/Assets/fixtures/Images.xcassets
-      Tests/Colors/fixtures/colors.txt
-      Tests/L10n/fixtures/Localizable.strings
-      Tests/Storyboard/fixtures/Message.storyboard
+      UnitTests/fixtures/Images.xcassets
+      UnitTests/fixtures/colors.txt
+      UnitTests/fixtures/Localizable.strings
+      UnitTests/fixtures/Message.storyboard
+      UnitTests/expected/Images-File-Defaults.swift.out
+      UnitTests/expected/Colors-File-Defaults.swift.out
+      UnitTests/expected/Strings-File-Defaults.swift.out
+      UnitTests/expected/Storyboards-Message-Defaults.swift.out
     ]
     pkgshare.install fixtures
-    pkgshare.install "Tests/Assets/expected/FileDefaults.swift.out" => "assets.swift"
-    pkgshare.install "Tests/Colors/expected/FileDefaults.swift.out" => "colors.swift"
-    pkgshare.install "Tests/L10n/expected/FileWithDefaults.swift.out" => "l10n.swift"
-    pkgshare.install "Tests/Storyboard/expected/MessageWithDefaults.swift.out" => "storyboard.swift"
   end
 
   test do
-    output = shell_output("#{bin}/swiftgen-assets #{pkgshare}/Images.xcassets").strip
-    assert_equal output, (pkgshare/"assets.swift").read.strip, "swiftgen-assets failed"
+    shell_output("#{bin}/swiftgen --version")
```

Comment:
`system` works exactly like `shell_output` except that it doesn’t capture the output; so that’s exactly what you’re looking for here :wink: 

It’s a wrapper, not the bare-bone Ruby `system`: https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/formula.rb#L1309